src/apps/companies/apps/advisers/client/ManageAdviser.jsx

```diff
@@ -25,71 +52,254 @@ CurrentLeadIta.propTypes = {
+                      required="Select an ITA"
+                      loadOptions={throttle(
+                        (searchString) =>
+                          axios
+                            .get('/api-proxy/adviser/', {
+                              params: {
+                                autocomplete: searchString,
+                              },
+                            })
+                            .then(({ data: { results } }) =>
+                              results
+                                .filter(
+                                  (adviser) => adviser?.name.trim().length
+                                )
+                                .map(({ id, name, dit_team }) => ({
+                                  label: `${name}${
+                                    dit_team ? ', ' + dit_team.name : ''
+                                  }`,
+                                  value: id,
+                                }))
+                            ),
+                        500
+                      )}
+                    />
+                    <H3 as="h2">What happens next</H3>
+                    <UnorderedList listStyleType="bullet">
+                      <ListItem>
+                        The lead ITA’s name and team will be shown on the
+                        company record page and on the lead ITA tab.
+                      </ListItem>
+                      <ListItem>
+                        This will replace all lead ITAs added on any
+                        subsidiaries of this company.
+                      </ListItem>
+                      <ListItem>
+                        Other ITAs can replace or remove this lead ITA at any
+                        time.
+                      </ListItem>
+                    </UnorderedList>
+                    <StyledStatusMessage>
+                      <p>
+                        <strong>
+                          When adding someone else as the Lead ITA, send the
+                          person an email to notify them. Data Hub does not send
+                          notications.
+                        </strong>
+                      </p>
+                    </StyledStatusMessage>
+                    <FieldInput
+                      type="hidden"
+                      name="companyId"
+                      initialValue={companyId}
+                    />
+
+                    <FormActions>
+                      <Button>Add Lead ITA</Button>
+                      <Link href={cancelUrl}>Cancel</Link>
+                    </FormActions>
+                  </Form>
+                </LoadingBox>
+              </Main>
+            </>
+          )
+        }}
+      </Task>
+    </>
+  ) : (
+    <>
+      <LocalHeader
+        heading={`${
+          currentLeadITA
+            ? 'Replace the Lead ITA'
+            : 'Confirm you are the Lead ITA'
+        }`}
```

Comment:
There is no need for template here
```suggestion
        heading={
          currentLeadITA
            ? 'Replace the Lead ITA'
            : 'Confirm you are the Lead ITA'
        }
```